Gemfile

```diff
@@ -3,8 +3,10 @@
 source 'https://rubygems.org'
 ruby IO.read(File.expand_path('.ruby-version', __dir__)).strip
 
+# autoprefixer-rails 10 or greater requires node.js 12 or greater
+gem 'autoprefixer-rails', '~> 9'
```

Comment:
This works because Bundler tries to be smarter than you, but:

* `'~> 1.1.1'` means 1.1.1 or 1.1.2 or 1.1.78, but not 1.2.x.
* `'~> 1.1'` means 1.1.x or 1.2.x or 1.100.x but not 2.x.y.
* By extension, you would think `'~> 9' would mean '9.x or 10.x or 67.x, but no version larger than infinity.

Luckily Bundler thinks that's dumb. `'~> 9'` and `'~> 9.0'` mean the same thing; but I prefer the one that follows the same pattern as the others.